### PR TITLE
migrate to postgres

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**

### CODE REVIEW

File: geo-data-server/src/main/java/org/gridsuite/geodata/server/repositories/LineRepository.java [hunk 1]
```diff
@@ -4,18 +4,26 @@
...
 import java.util.List;
 
 /**
  * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
  */
 @Repository
-public interface LineRepository extends CassandraRepository<LineEntity, String> {
+public interface LineRepository extends JpaRepository<LineEntity, String> {
 
     List<LineEntity> findByCountry(String country);
```
Comment (FranckLecuyer) [inline geo-data-server/src/main/java/org/gridsuite/geodata/server/repositories/LineRepository.java:22]: seems not being used anymore

File: README.md [hunk 2]
```diff
@@ -6,15 +6,11 @@
...
-$ mvn verify
-```
-
-OR
 
-```bash
-$ mvn verify -DproxyHost=proxy.com -DproxyPort=8080 -DproxyUser=user -DproxyPassword=XXXX
+The old way to automatically generate the sql schema file (directly using hibernate) can still be used for debugging. Use the following command:
+```
+mvn package -DskipTests && rm -f src/main/resources/geoData.sql && java  -jar target/gridsuite-study-server-1.0.0-SNAPSHOT-exec.jar --spring.jpa.properties.javax.persistence.schema-generation.scripts.action=create
```
[the remark this answers is not in the document]
Comment (JacquesBorsenberger) [reply README.md]: done

File: geo-data-server/src/main/java/org/gridsuite/geodata/server/repositories/LineRepository.java (hunk 1 above)
Comment (JacquesBorsenberger) [reply geo-data-server/src/main/java/org/gridsuite/geodata/server/repositories/LineRepository.java]: removed

File: geo-data-server/src/main/java/org/gridsuite/geodata/server/repositories/SubstationRepository.java [hunk 3]
```diff
@@ -6,16 +6,19 @@
...
 import java.util.List;
 
 /**
  * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
  */
 @Repository
-public interface SubstationRepository extends CassandraRepository<SubstationEntity, String> {
+public interface SubstationRepository extends JpaRepository<SubstationEntity, String> {
 
     List<SubstationEntity> findByCountry(String country);
```
[the remark this answers is not in the document]
Comment (JacquesBorsenberger) [reply geo-data-server/src/main/java/org/gridsuite/geodata/server/repositories/SubstationRepository.java]: removed

**Review by FranckLecuyer — APPROVED** [review]

**Review by laurgarn — APPROVED** [review]

**Review by jonenst — APPROVED** [review]

**Pull request merged**